src/screens/VisualTests/StoryInfo.tsx

```diff
@@ -133,12 +133,14 @@ export const StoryInfo = ({
   } else if (shouldSwitchToLastBuildOnBranch) {
     details = (
       <Info>
-        <b>
-          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-          <Link isButton onClick={switchToLastBuildOnBranch}>
-            View latest snapshot
-          </Link>
-        </b>
+        <span>
+          <b>
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+            <Link isButton onClick={switchToLastBuildOnBranch}>
+              View latest snapshot
+            </Link>
+          </b>
+        </span>
```

Comment:
Was the console error that <b> could not be nested directly withing <Info>?

Reply:
No this was about the `:first-child` selector being vulnerable in SSR situations (hence not relevant, but a bad look nonetheless). I replaced it with `span:first-of-type` but that only works if they're all spans. This one wasn't and so I wrapped it.